module/VuFind/src/VuFind/View/Helper/Root/Citation.php

```diff
@@ -192,6 +192,47 @@ public function getCitation($format)
         return '';
     }
 
+    protected function trimPunctuation($text)
+    {
+        return trim($text, " \n\r\t\v\0,:;/");
+    }
+
+    protected function removeDateRange($name)
```

Comment:
Can you use the existing cleanNameDates function instead?

Reply:
`cleanNameDates` didn't cover all the cases. I do intend to clean unused functions out when the new config is more integrated.

Reply:
I'd be interested to know what cases aren't covered, because I think this is exactly the intended purpose for `cleanNameDates`. Maybe we need to fix that one rather than add a second function.